home.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>My component library</title>
+    <script src="https://kit.fontawesome.com/917a992f26.js" crossorigin="anonymous"></script>
+    <link href="./styles.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@400;900&display=swap" rel="stylesheet">
+</head>
+<body>
+<header>
+    <a>Login</a>
+    <a>Signin</a>
```

Comment:
wrap this in a nav

Reply:
well home.html is not ready, just sign in and signup